PrincessBrideTrivia/PrincessBrideTrivia.Tests/ProgramTests.cs

```diff
@@ -72,6 +72,22 @@ public void GetFilePath_ReturnsFileThatExists()
             Assert.AreEqual(expectedString, percentage);
         }
 
+        [TestMethod]
+        public void RandomizeQuestions_GivenValidArray_ReturnsRandomizedArray()
+        {
+            //Arrange
+            Question[] questions = new Question[3];
+            Question[] questionsCpy = new Question[questions.Length];
```

Comment:
```suggestion
            Question[] questionsCopy = new Question[questions.Length];
```

Not sure if this is a typo, but definitely make sure to spell out words rather than abbreviate in the majority of cases.